gui/builtinViewColumns/state.py

```diff
@@ -68,11 +68,17 @@ def getImageId(self, stuff):
                                                                 "gui")
         elif isinstance(stuff, Fit):
             fitID = self.mainFrame.getActiveFit()
-            projectionInfo = stuff.getProjectionInfo(fitID)
 
-            if projectionInfo is None:
+            # can't use isinstance here due to being prevented from importing CommandView.
+            #  So we do the next best thing and compare Name of class.
+            if type(self.fittingView).__name__ == "CommandView":
```

Comment:
Could use:
`self.fittingView.__class__.__name__`

Not sure it's any faster though.